src/renderer/containers/ReplayFileStats/OverallTable.tsx

```diff
@@ -0,0 +1,403 @@
+import _ from "lodash";
+import React from "react";
+import { FileResult } from "../../../common/replayBrowser/types";
+import { extractPlayerNames } from "common/matchNames";
+import { RatioType, StatsType } from "@slippi/slippi-js";
+import { getCharacterIcon } from "@/lib/utils";
+import * as T from "./TableStyles";
+
+const columnCount = 5; // Unfortunately there is no way to specify a col span of "all" max cols there will be is 5
+
+export interface OverallTableProps {
+  file: FileResult;
+  stats: StatsType | null;
+}
+
+export const OverallTable: React.FC<OverallTableProps> = ({ file, stats }) => {
+  if (!stats) return <div>An Error Occurred!</div>;
```

Comment:
`stats` should not be nullable. If `stats` is null, the parent should render an error message and the child components should never have to handle the null case.